modules/ossm-config-disable-networkpolicy.adoc

```diff
@@ -0,0 +1,40 @@
+////
+This module included in the following assemblies:
+-v2x/servicemesh-release-notes.adoc
+////
+
+[id="ossm-config-disable-networkpolicy_{context}"]
+= Disabling network policies
```

Comment:
The content here is good, though when I look at the preview, it shows up at the end of the very long set of release notes, rather than being under the 2.1.1 heading. There probably needs to be a one liner about this change under the 2.1.1 heading in addition the usual "CVEs and bug fixes" sentence. This content is fairly detailed though, and should more likely be consolidated with [this change](https://github.com/openshift/openshift-docs/pull/40095) in the Preparing for the Installation section. Alternatively, it could also be in the Service Mesh configuration section.

Reply:
@longmuir   Jonh's PR put this content as part of supported configurations, which really isn't the right place for it.  We need to figure out a better place to put this procedure.  Putting in the Release Notes is a short term fix until we find a better place to put this in the doc set.

Reply:
Ok - but, at least find a better place than buried under all of the release notes (ie under the 2.1.1 header). As this will be introduced in 2.1.1 (and won't apply to < 2.1), that should also be made clear.